x-pack/plugins/apm/public/context/url_params_context/helpers.test.ts

```diff
@@ -175,10 +166,10 @@ describe('url_params_context helpers', () => {
     });
 
     ['s', 'm', 'h', 'd', 'w'].map((roundingOption) =>
-      it(`removes /${roundingOption} rounding option from relative time`, () => {
+      it(`doesn't remove /${roundingOption} rounding option when time is not added or subtracted`, () => {
         const spy = jest.spyOn(datemath, 'parse');
         helpers.getExactDate(`now/${roundingOption}`);
-        expect(spy).toHaveBeenCalledWith('now', {});
+        expect(spy).toHaveBeenCalledWith(`now/${roundingOption}`, {});
       })
```

Comment:
Please double check that this is the behaviour we want. And would be great with more test coverage